__tests__/games.test.js

```diff
@@ -170,4 +170,64 @@ describe("PATCH: /api/reviews/:review_id", () => {
             expect(body.msg).toBe("bad request")
         })
     })
+})
+
+describe("GET: /api/reviews/:review_id", () => {
+    it("200: should respond with an object of review of correct id with a comment_count", () => {
+        return request(app)
+        .get("/api/reviews/3")
+        .expect(200)
+        .then(({body}) => {
+            expect(typeof body.review).toBe("object")
+            expect(body.review).toMatchObject({
+                    title: 'Ultimate Werewolf',
+                    designer: 'Akihisa Okui',
+                    owner: 'bainesface',
+                    review_img_url:
+                      'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                    review_body: "We couldn't find the werewolf!",
+                    category: 'social deduction',
+                    created_at: expect.any(String),
+                    votes: 5,
+                    comment_count: 3
```

Comment:
I'm not going to make you change this if you don't want to, but it's worth noting we want to avoid editing old tests to include new properties. We would usually just add a new test for that new property